### PR TITLE
[ #1417] improve(doc): Check getting started document

### PR DESCRIPTION
**Pull request by hiirrxnn**
What changes were proposed in this pull request?
Improved Getting Started document

Why are the changes needed?
For clear and correct instructions

Fix: #1417 

Does this PR introduce any user-facing change?
N/A

How was this patch tested?
N/A.

### CODE REVIEW

**Comment by hiirrxnn** [comment]
Please review

File: docs/getting-started.md [hunk 1]
```diff
@@ -21,11 +21,44 @@ or locally see [Installing Gravitino playground locally](#installing-gravitino-p
 
 If you are using AWS and want to access the instance remotely, be sure to read [Accessing Gravitino on AWS externally](#accessing-gravitino-on-aws-externally)
 
+### Key Sections:
+
+1. **Installation Methods:**
```
Comment (justinmclean) [inline docs/getting-started.md:26]: I would remove the ":" from all the headings

File: docs/getting-started.md [hunk 2]
```diff
@@ -21,11 +21,44 @@ or locally see [Installing Gravitino playground locally](#installing-gravitino-p
 
 If you are using AWS and want to access the instance remotely, be sure to read [Accessing Gravitino on AWS externally](#accessing-gravitino-on-aws-externally)
 
+### Key Sections:
```
Comment (justinmclean) [inline docs/getting-started.md:24]: "Key Sections" is probably not quite correct here, perhaps use "Index" instead.

File: docs/getting-started.md [hunk 3]
```diff
@@ -21,11 +21,44 @@ or locally see [Installing Gravitino playground locally](#installing-gravitino-p
...
+
+8. **Using REST to Interact with Gravitino:**
+   - Examples of interacting with Gravitino via REST commands, demonstrating how to create and modify metadata.
+
+9. **Accessing Gravitino on AWS Externally:**
+   - Guidelines for accessing Gravitino externally when deployed on AWS, including necessary configurations and considerations.
+
+10. **Next Steps and Conclusion:**
+    - Concluding thoughts and suggesting next steps for users who have completed the setup.
+
```
Comment (justinmclean) [inline docs/getting-started.md:55]: This section doesn't contain a conclusion.

File: docs/getting-started.md [hunk 4]
```diff
@@ -21,11 +21,44 @@ or locally see [Installing Gravitino playground locally](#installing-gravitino-p
 
 If you are using AWS and want to access the instance remotely, be sure to read [Accessing Gravitino on AWS externally](#accessing-gravitino-on-aws-externally)
 
+### Key Sections:
+
+1. **Installation Methods:**
+   - Explore different installation methods, from using Docker to setting up Gravitino on cloud platforms or locally.
+
+2. **Java Development Kit (JDK):**
```
Comment (justinmclean) [inline docs/getting-started.md:29]: We follow the Google Developer Guide for documents and that uses sentence case, so we'll need to remove the capitalisation here and elsewhere.

File: docs/getting-started.md [hunk 5]
```diff
@@ -183,7 +216,8 @@ Gravitino provides a bundle of Docker images to launch a Gravitino playground, w
 includes Apache Hive, Apache Hadoop, Trino, MySQL, PostgreSQL, and Gravitino. You can use
 Docker compose to start them all.
 
-Installing Docker and Docker Compose is a requirement for using the playground.
+Installing Docker and Docker Compose is a requirement for using the playground. 
+Note : The following command is more suitable for Debian/Ubuntu systems. Users on other systems (like Fedora, CentOS) should use the appropriate package manager.
```
Comment (justinmclean) [inline docs/getting-started.md:220]: No need for this Note here given the context i.e. installing on Ubuntu.

File: docs/getting-started.md [hunk 6]
```diff
@@ -282,3 +316,27 @@ webserver.host` parameter from `127.0.0.1` to your AWS instance's private IP4 ad
 You'll also need to open port 8090 in the security group of your AWS instance to access Gravitino. To access Hive you need to open port 10000 in the security group.
 
 After completing these steps, you should be able to access the Gravitino REST interface from either the command line or a web browser on your local computer. You can also connect to Hive via DBeaver or any other database IDE.
+
+## Conclusion and Next Steps
+
+Congratulations! You've successfully set up Gravitino on your chosen platform. Here's a quick summary of key points and some next steps:
```
Comment (justinmclean) [inline docs/getting-started.md:322]: I would remove this as it's not a conclusion.

File: docs/getting-started.md [hunk 7]
```diff
@@ -282,3 +316,27 @@ webserver.host` parameter from `127.0.0.1` to your AWS instance's private IP4 ad
...
+
+### Next Steps:
+
```
Comment (justinmclean) [inline docs/getting-started.md:325]: Would be nice to link each of these sections to other documentation.

File: docs/getting-started.md [hunk 8]
```diff
@@ -282,3 +316,27 @@ webserver.host` parameter from `127.0.0.1` to your AWS instance's private IP4 ad
...
+1. **Explore Documentation:**
+   - Delve deeper into the Gravitino documentation for advanced features, configuration options, and troubleshooting tips.
+
+2. **Community Engagement:**
+   - Join the Gravitino community forums or mailing lists to connect with other users, share experiences, and seek assistance if needed.
```
Comment (justinmclean) [inline docs/getting-started.md:330]: We don't have a mailing list so we should limit it to things that currently exist.

File: docs/getting-started.md [hunk 9]
```diff
@@ -282,3 +316,27 @@ webserver.host` parameter from `127.0.0.1` to your AWS instance's private IP4 ad
...
 After completing these steps, you should be able to access the Gravitino REST interface from either the command line or a web browser on your local computer. You can also connect to Hive via DBeaver or any other database IDE.
+
+## Conclusion and Next Steps
+
+Congratulations! You've successfully set up Gravitino on your chosen platform. Here's a quick summary of key points and some next steps:
+
+### Next Steps:
+
+1. **Explore Documentation:**
+   - Delve deeper into the Gravitino documentation for advanced features, configuration options, and troubleshooting tips.
```
Comment (justinmclean) [inline docs/getting-started.md:327]: We currently don't have any toubleshooting tips.

File: docs/getting-started.md [hunk 10]
```diff
@@ -282,3 +316,27 @@ webserver.host` parameter from `127.0.0.1` to your AWS instance's private IP4 ad
...
+
+2. **Community Engagement:**
+   - Join the Gravitino community forums or mailing lists to connect with other users, share experiences, and seek assistance if needed.
+
+3. **Custom Configurations:**
+   - Tailor Gravitino to your specific needs by exploring and adjusting configurations in the `gravitino.conf` file.
```
Comment (justinmclean) [inline docs/getting-started.md:333]: This repeats a bit of item 1

File: docs/getting-started.md [hunk 11]
```diff
@@ -282,3 +316,27 @@ webserver.host` parameter from `127.0.0.1` to your AWS instance's private IP4 ad
...
+
+4. **Integration with Other Tools:**
+   - Consider integrating Gravitino with other tools or databases to enhance your data processing capabilities.
+
+5. **Continuous Updates:**
+   - Stay informed about Gravitino updates and new releases to benefit from the latest features, optimizations, and security enhancements.
```
Comment (justinmclean) [inline docs/getting-started.md:339]: How?

File: docs/getting-started.md [hunk 12]
```diff
@@ -282,3 +316,27 @@ webserver.host` parameter from `127.0.0.1` to your AWS instance's private IP4 ad
...
+
+Remember, this document serves as a starting point. Feel free to customize your Gravitino setup based on your requirements and explore the vast possibilities this powerful tool offers. If you encounter any issues or have questions, don't hesitate to reach out to the Gravitino community for support.
```
Comment (justinmclean) [inline docs/getting-started.md:341]: This sounds a little too much like ChapGTP, probably best to reword. Again linking to how people can reach out would be helpful.

**Review by justinmclean — COMMENTED** [review]
Thanks for the contribution, I left some comments on how this could be improved.

File: docs/getting-started.md (hunk 7 above)
Comment (hiirrxnn) [reply docs/getting-started.md:326]: Could you give me an example ?

File: docs/getting-started.md (hunk 11 above)
Comment (hiirrxnn) [reply docs/getting-started.md:331]: Should i remove those points altogether?

**Comment by hiirrxnn** [comment]
If you want a specific thing to be added in the doc , that I haven't added ,please let me know .

File: docs/getting-started.md (hunk 11 above)
Comment (justinmclean) [reply docs/getting-started.md:331]: I wouldn't remove them, but you need to explain how someone can stay informed.

Comment (hiirrxnn) [reply docs/getting-started.md:331]: Could you provide me with the relevant information?

**Comment by hiirrxnn** [comment]
What should I do about the one failing test?

File: docs/getting-started.md (hunk 7 above)
Comment (justinmclean) [reply docs/getting-started.md:326]: For example link to https://datastrato.ai/docs in step 1.

File: docs/getting-started.md (hunk 11 above)
Comment (justinmclean) [reply docs/getting-started.md:331]: How would someone stay informed? You need to give details on what they should do.

File: docs/getting-started.md [hunk 13]
```diff
@@ -1,7 +1,7 @@
 ---
 title: "Getting started with Gravitino"
 slug: /getting-started
-license: "Copyright 2023 Datastrato Pvt Ltd.
+license: "Copyright 2023 Datastrato Pvt. Ltd.
```
Comment (justinmclean) [inline docs/getting-started.md:4]: Changing this line caused the license check to fail, please undo this change.

**Comment by jerryshao** [comment]
@justinmclean do you have further comment?

**Review by jerryshao — APPROVED** [review]
LGTM.
